onnxruntime/python/tools/transformers/models/sam2/image_encoder.py

```diff
@@ -113,17 +114,65 @@ def export_image_encoder_onnx(
         if not verbose:
             warnings.filterwarnings("ignore", category=torch.jit.TracerWarning)
             warnings.filterwarnings("ignore", category=UserWarning)
-        torch.onnx.export(
-            sam2_encoder,
-            image,
-            onnx_model_path,
-            export_params=True,
-            opset_version=17,
-            do_constant_folding=True,
-            input_names=["image"],
-            output_names=["image_features_0", "image_features_1", "image_embeddings"],
-            dynamic_axes=dynamic_axes,
-        )
+
+        if not dynamo:
```

Comment:
nit: do the other way `if dynamo:`

Reply:
I prefer to put the long one in else so that if and else are close when viewing code.